src/app/components_ngrx/work-item-quick-add/work-item-quick-add.component.ts

```diff
@@ -74,7 +75,8 @@ export class WorkItemQuickAddComponent implements OnInit, OnDestroy, AfterViewIn
     private route: ActivatedRoute,
     private renderer: Renderer2,
     private store: Store<AppState>,
-    private workItemQuery: WorkItemQuery) {}
+    private workItemQuery: WorkItemQuery,
+    private filterService: FilterService) {}
```

Comment:
Can we use iterationQuery instead? You can get the selected iteration from there.

Reply:
I havent added workItemQuery and it is already being used `this.workItemQuery.getWorkItems()
        .filter(items => !!items.length)
        .subscribe(items => {
          // const addedItem = items.find(item => item.createId === this.createId);
          this.resetQuickAdd();
        })`

Reply:
I want the iteration id of the iteration the user clicks on and not the selected iteration.

Reply:
Selected iteration is one among the active iterations if a I use that all the newly created wis will be added to one selected iteration and not the iteration the user clicked on. I can change the variable name to something like `openIterationID` if  `selectedIterationID` is confusing.